src/backend/optimizer/plan/subselect.c

```diff
@@ -1468,6 +1456,29 @@ convert_ANY_sublink_to_join(PlannerInfo *root, SubLink *sublink,
 	if (!bms_is_subset(upper_varnos, available_rels))
 		return NULL;
 
+	/*
+	 * GPDB_92_MERGE_FIXME:
```

Comment:
Does this need a FIXME comment once merged? Are we leaving any known open items from this PR?

Reply:
I believe there is no known issues from this PR, but not with full confidence. Will remove the FIXEME once this PR is reviewed.